pbsmetrics/config/metrics.go

```diff
@@ -6,9 +6,9 @@ import (
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	"github.com/prebid/prebid-server/pbsmetrics"
-	"github.com/prebid/prebid-server/pbsmetrics/prometheus"
-	"github.com/rcrowley/go-metrics"
-	"github.com/vrischmann/go-metrics-influxdb"
+	prometheusmetrics "github.com/prebid/prebid-server/pbsmetrics/prometheus"
+	metrics "github.com/rcrowley/go-metrics"
+	influxdb "github.com/vrischmann/go-metrics-influxdb"
```

Comment:
what made the imports change? not that it's a big deal.. things still compile so I assume it still works.

Reply:
That is a good question, I did not consciously make this change.

Line 40 below shows that the old prometheus import was being accessed by the label `prometheusmetrics` before the change, so I am guessing new linting in either upgraded go package or VSCode now triggers the move to make inobvious module names explicit.